contracts/cheddar_5x5_tic_tac_toe/src/internal.rs

```diff
@@ -109,14 +108,19 @@ impl Contract {
             .unwrap_or(0)
             .checked_mul(self.service_fee as u128)
             .unwrap_or(0);
-        assert!(fees_amount > 0, "Incorrect fees computing");
+        // assert!(fees_amount > 0, "Incorrect fees computing"); //TODO: i think we do not need this assertion since we allow zero fees
 
         let winner_reward: Balance = players_deposit.0 - fees_amount;
 
         if let Some(winner_id) = winner {
             log!("Winner is {}. Reward: {}", winner_id, winner_reward);
-
-            self.internal_transfer(&token_id, winner_id, winner_reward.into());
+            let stats = self.get_stats(winner_id);
+            self.internal_transfer(&token_id, winner_id, winner_reward.into())
```

Comment:
maybe we should also set a static GAS here?

Reply:
not sure. Could you provide more details why we would want to do it?

Reply:
in the `internal_transfer` let's also reserve gas, to make sure we have enough gas for `transfer_callback`.

Reply:
we are doing it inside of `internal_transfer`